.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -161,15 +161,14 @@ jobs:
         unzip -l "$path"
         echo "STACKROX_NVD_ZIP_PATH=$path" >> "$GITHUB_ENV"
 
-    - name: Run Updater (single bundle)
```

Comment:
I think 4.4 used the single bundle. Yes, it's out of support and it was only tech preview, but we still have not set the precedent halting vuln updates for old versions. So, I'm inclined to say we should keep this. We can probably update the ordering, though, and maybe upload and delete this file once we are done here before moving onto the multi bundle. What do you think?

Reply:
I'm not _super_ familiar with the updater, but I thought this change would resolve your concern: https://github.com/stackrox/stackrox/pull/15874/files#diff-f68747960bca95fedc4229de2430278fd5a4ffb60e71c4cd2cb5c016542d663fR169-R171

Let me know if that doesn't work, though (and a brief explanation, please 🙂).